components/common/posts/ImageList.tsx

```diff
@@ -0,0 +1,53 @@
+import { AppBskyEmbedImages } from '@atproto/api'
+import { useState } from 'react'
+import { InformationCircleIcon } from '@heroicons/react/24/outline'
+import Lightbox from 'yet-another-react-lightbox'
+import Captions from 'yet-another-react-lightbox/plugins/captions'
+import Thumbnails from 'yet-another-react-lightbox/plugins/thumbnails'
+import 'yet-another-react-lightbox/styles.css'
+import 'yet-another-react-lightbox/plugins/thumbnails.css'
+import 'yet-another-react-lightbox/plugins/captions.css'
```

Comment:
I see these CSS imports a couple places.  Can/should we hoist them up near the entrypoint rather than pull them in wherever `yet-another-react-lightbox` is used?

Reply:
oh the other places I included it were leftovers from initial implementation. I was thinking including it in the component would be better so that we are not loading these on initial landing and once you visit a screen where the component is rendered, the css would be cached and future requires would just get cache hit.
however, these are very lightweight, like 3kb each so not too worried about it the load time either way. ended up including it in layout file.